s3prl/downstream/specaug.py

```diff
@@ -208,10 +227,23 @@ def mask_along_axis(self, spec, spec_lengths):
         B = spec.shape[0]
         # D = Length or Freq
         D = spec.shape[self.dim]
+        T = self.mask_width_range[1]
+        # Adaptive_SpecAugment
+        if self.dim == 1 & self.adaptive :
+          if self.adaptive_number_ratio > 0:
+            self.num_mask = min(int(self.adaptive_number_ratio * D), self.max_n_time_masks)
+          else:
+            self.num_mask = num_mask
```

Comment:
Do you mean line 235-236? It seems like line 235-236 is redundant, you can just simply remove it. I just want to make sure that adaptive number ratio is greater than zero, otherwise we do not change the number of masks. Also, I am a little confused about the difference between `self.num_mask `and `num_mask` as you mentioned below.